### DISCUSSION TITLE
empty feed_infos

### DISCUSSION THREAD

**Discussion by boredland** [transitland/transitland#368] [Q&A] [answered]
Hi there! We realized, that there are `feed_version`s that do not contain feed_infos (empty array).

```gql
{
  stops(limit: 10, where: {onestop_id: "s-u1p5dv4fr7-badsooden~allendorf"}) {
    id
    onestop_id
    feed_version {
      fetched_at
      feed_infos {
        feed_publisher_name
        feed_publisher_url
      }
    }
  }
}
```

drilling down, this stop seems to originate from the feed [f-nvbw~bwspnv](https://www.transit.land/feeds/f-nvbw~bwspnv), which seems to miss that information.

Could you explain to us why that metadata is missing and if / how we could contribute that? In this particular case it seems to be [nvbw](https://www.nvbw.de/).

**Comment by drewda** [comment, marked as answer]
Hi @boredland, in this case the source feed does not provide a `feed_info.txt` file.

Here is a list of the files that were found and imported from that feed and feed version: https://www.transit.land/feeds/f-nvbw~bwspnv/versions/bdefe6693816f3536418cca4d6420b0721a4f31d#files

If you have a contact at nvbw.de, it would be great to add them to add a `feed_info.txt` file. Here is more information about the potential contents for that file: https://gtfs.org/schedule/reference/#feed_infotxt

**Reply by boredland** [reply]
Hi @drewda! Thanks for the hint. We can try to get in touch with them!

**Comment by wiese** [comment]
Anyone interested in reporting problems with GTFS data may find the following repository helpful: https://github.com/mfdz/GTFS-Issues/
